### PR TITLE
Release notes for 5.11.0

### PR DESCRIPTION
**Pull request by velzevur**
Add missing release notes.

The work on this PR had been supported by ACF.

### CODE REVIEW

**Review by uwiger — APPROVED** [review]

File: docs/release-notes/RELEASE-NOTES-5.11.0.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+It:
+
+* Fixes an issue where some nodes are stuck and could not sync beyond a
+  certain point.
+
+* Exports a function (`aec_chain_state:grant_fees/5`) in order to be used in
+  the MDW while doing dry runs.
+
+* Improves the transaction pool to not allow in `paying_for` transactions that
```
Comment (hanssv) [inline docs/release-notes/RELEASE-NOTES-5.11.0.md:13]: Well until Iris the Tx-pool will reject them anyway :-)

**Review by hanssv — APPROVED** [review]

File: docs/release-notes/RELEASE-NOTES-5.11.0.md (hunk 1 above)
Comment (velzevur) [reply docs/release-notes/RELEASE-NOTES-5.11.0.md:13]: Indeed but it felt like wrong adding it to `iris` release notes anyway

**Review by radrow — APPROVED** [review]

**Comment by velzevur** [comment]
Will have to accommodate changes from #3565

**Comment by velzevur** [comment]
Missing release line in #3583

**Review by gorbak25 — APPROVED** [review]

**Review by gorbak25 — APPROVED** [review]

**Review by uwiger — APPROVED** [review]

File: docs/release-notes/RELEASE-NOTES-5.11.0.md (hunk 1 above)
Comment (radrow) [reply docs/release-notes/RELEASE-NOTES-5.11.0.md:13]: I believe you always say "not to" instead of "to not"

**Pull request merged**